src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,31 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: 728px;
+  margin: 0 auto;
+  padding: 10px;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    margin-block-end: $margin-day;
+    margin-inline-end: $margin-day;
+    width: $size-day;
+    height: $size-day;
+    border: 1px solid #000;
+    background-color: #eee;
+    transition-duration: 0.5s;
```

Comment:
It would be great to add transition properties too. Now it applies to all properties, which is not good for performance
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x
Also, let's make transition duration a variable. Same with all colors